egs/zeroth_korean/s5/run.sh

```diff
@@ -0,0 +1,194 @@
+#!/bin/bash
+#
+# Based mostly on the WSJ/Librispeech recipe. The training database is #####,
+# it consists of 51hrs korean speech with cleaned automatic transcripts:
+#
+# http://www.openslr.org/resources (Mirror).
+#
+# Copyright  2018  Atlas Guide (Author : Lucas Jo)
+#            2018  Gridspace Inc. (Author: Wonkyum Lee)
+#
+# Apache 2.0
+#
+
+# Check list before start
+# 1. locale setup
+# 2. pre-installed package: awscli, Morfessor-2.0.1, flac, sox, same cuda library, unzip
+# 3. pre-install or symbolic link for easy going: rirs_noises.zip (takes pretty long time)
+# 4. parameters: nCPU, num_jobs_initial, num_jobs_final, --max-noises-per-minute
+
+db_dir=./db
+nCPU=16
+
+. ./cmd.sh
+. ./path.sh
+
+# you might not want to do this for interactive shells.
+set -e
+
+startTime=$(date +'%F-%H-%M')
+echo "started at" $startTime
+
+# download the data.  
+local/download_and_untar.sh $db_dir
+
+# format the data as Kaldi data directories
+for part in train_data_01 test_data_01; do
+	# use underscore-separated names in data directories.
+	local/data_prep.sh $db_dir/$part data/$(echo $part | sed s/-/_/g)
+done
+
+# update segmentation of transcripts
+for part in train_data_01 test_data_01; do
+	local/updateSegmentation.sh data/$part data/local/lm
+done
+
+# prepare dictionary and language model 
+local/prepare_dict.sh data/local/lm data/local/dict_nosp
+
+utils/prepare_lang.sh data/local/dict_nosp \
+	"<UNK>" data/local/lang_tmp_nosp data/lang_nosp
+
+local/format_lms.sh --src-dir data/lang_nosp data/local/lm
+
+# Create ConstArpaLm format language model for full 3-gram and 4-gram LMs
+# it takes long time and do this again after computing silence prob.
+# you can do comment out here this time
+
+#utils/build_const_arpa_lm.sh data/local/lm/zeroth.lm.tg.arpa.gz \
```

Comment:
if this is part of the script it shouldn't be commented out.  You can add a --state option in the run.sh to enable running after partial run.